src/ZoneServer/Network/Send.cs

```diff
@@ -4474,5 +4484,17 @@ public static void ZC_ACTION_PKS(IActor toActor, IActor fromActor, byte type, in
 
 			toActor.Map.Broadcast(packet);
 		}
+
+		/// Cancel "timed action" due to mouse movement (walking)
```

Comment:
There seems to be a `<summary>` missing here. Also, could you clarify what exactly this does? The way you're using it, it doesn't seem related to mouse movement, and it's unclear to me what `walking` refers to in this context.